tests/upgrades/test_performance_tuning.py

```diff
@@ -141,18 +144,24 @@ def test_pre_performance_tuning_apply(self):
         mongodb_type = ssh.command(cmd).return_code
         self._create_custom_hiera_file(mongodb_type, "medium")
         try:
-            ssh.upload_file('custom-hiera.yaml', '/etc/foreman-installer')
+            ssh.upload_file(
+                local_file='custom-hiera.yaml',
+                remote_file='/etc/foreman-installer/custom-hiera.yaml',
+            )
             command_output = ssh.command(
-                'satellite-installer -s --disable-system-checks', connection_timeout=1000
-            ).stdout
-            assert '  Success!' in command_output
+                'satellite-installer -s --disable-system-checks', timeout=1000
+            )
+            assert '  Success!' in command_output.stdout
```

Comment:
why extra space in '  Success!' ?

`assert 'Success!' in command_output.stdout` also work